benchmarks/benchmark.cpp

```diff
@@ -1,6 +1,6 @@
+#include "event_counter.h"
```

Comment:
Move this after the ifdef in order to use hardware counters for Mac and Linux

Reply:
Done.